src/strategies/eth-wallet-age/examples.json

```diff
@@ -0,0 +1,33 @@
+[
+  {
+    "name": "Age in Days",
+    "strategy": {
+      "name": "eth-wallet-age",
+      "params": {
+        "dfuseApiKey": "web_f527db575a38dd11c5b686d7da54d371"
```

Comment:
I think this API Key should be hardcoded within the strategy. Is there some limitation with the API?